tools/azsdk-cli/docs/specs/0-scenario-1.spec.md

```diff
@@ -233,31 +236,43 @@ Check if my environment is ready to generate SDKs for all languages.
 
 **Prompt:**
 ```
-Generate the Health Deidentification SDK for all languages from the TypeSpec in azure-rest-api-specs.
+Use MCP tool `azsdk_package_generate_code` to generate the Health Deidentification SDK for all languages from the TypeSpec in azure-rest-api-specs. Then use MCP tool to build the generated SDK code.
```

Comment:
Non-blocking nit: Do we think users are actually going to mention the tools by name? I think it's more likely they'll just ask for the libraries to be generated.

Reply:
That's true. This prompt is only for the case we don't have the instructions in place. Currently we have another SDK generation and release workflow documented in the instruction files of public spec repo. User has to call out the tool name so that the agent can run the correct tools.

Later on, user shouldn't need to mention `MCP` at all when we update the instruction files to include all the end-to-end tools.